en/engine/event/event-node.md

```diff
@@ -0,0 +1,244 @@
+|:-------------------------------|:---------------------------------------------------------------------------------------------|
+| __Node.EventType.MOUSE_DOWN__  | When a button on the mouse is pressed in the target node region                              |
+| __Node.EventType.MOUSE_ENTER__ | When the cursor enters the target node region, whether or not the button is pressed          |
+| __Node.EventType.MOUSE_MOVE__  | When the cursor is moved in the target node region, whether or not the button is pressed     |
+| __Node.EventType.MOUSE_LEAVE__ | When the cursor is moved out of the target node region, whether or not the button is pressed |
+| __Node.EventType.MOUSE_UP__    | When a button on the mouse is released                                                       |
+| __Node.EventType.MOUSE_WHEEL__ | When the mouse wheel is scrolled                                                             |
+
+The important APIs of mouse events (`Event.EventMouse`) are described in the [Mouse Events API](event-api.md#Mouse-Event-API) (`Event` standard events API excluded).
+
+## Touch event types and event objects
+
+__Touch events__ can be triggered on both mobile platforms and desktop platforms. Developers can better debug on the desktop platform, by simply listening for touch events and responding to both mobile touch events and desktop mouse events at the same time. The types of touch events provided by the system are as follows:
+
+| Enumerated Object Definition  | Timing of Event Triggering                                                               |
+|:--------------------------------|:-----------------------------------------------------------------------------------------|
+| __Node.EventType.TOUCH_START__  | When one or more touch points are placed in the target node region                       |
+| __Node.EventType.TOUCH_MOVE__   | When one or more touch points are moved along the screen                                 |
+| __Node.EventType.TOUCH_END__    | When one or more touch points are removed from the screen in the target node region      |
+| __Node.EventType.TOUCH_CANCEL__ | When one or more touch points are removed from the screen outside the target node region |
+
+The important APIs of a touch event (`Event.EventTouch`) are described in the [Mouse Events API](event-api.md#Touch-Event-API) (`Event` standard event API excluded):
+
+> __Note__: touch events support multi-touch, each touch spot will send one event to the event listener.
+
+## Node Event Dispatching
+
+We support `dispatchEvent` interface on `Node`. Events dispatched by this interface, would enter the event delivery stage. The event dispatching system of Cocos Creator is based on the implementation of [event bubbling and capture on Web standard](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events#event_bubbling_and_capture). After the event is dispatched, it will go through the following three stages:
```

Comment:
```suggestion
The `dispatchEvent` interface is supported on `Node`. Events dispatched by this interface, would enter the event delivery stage. The event dispatching system of Cocos Creator is based on the implementation of [event bubbling and capture on Web standard](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events#event_bubbling_and_capture). After the event is dispatched, it will go through the following three stages:
```